drivers/ksz8851snl/include/ksz8851snl_config.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2015
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    driver_ksz8851snl ksz8851snl
+ * @ingroup     drivers
+ * @brief       Driver for the ksz8851snl Ethernet Adapter
+ * @{
+ *
+ * @file
+ * @brief       ksz8851snl Configuration File
+ *
+ * @author      Matt Poppe <matt@poppe.me>
+ */
+
+
+/*
+ * TODO: Currently placed inside driver file just for initial convenance will
+ * Get moved to the board folder in the future
```

Comment:
Actually, not sure if the board folder is a good place for this -> so far we have only means of mapping fixed connected devices on the board. @kaspar030: Could you think of a way, how we could map optinally connected devices with fixed pin config (e.g. standardized connectors) into the current board auto_init concept?

@poppe34 You could take a look at `auto_init` w.r.t. network interfaces, e.g. `sys/auto_init/netif/auto_init_at86rf2xx.c` to see how the auto-initialization of network devices currently works. Let us know if we can help!